### DISCUSSION TITLE
Não acertar a respostas.

### DISCUSSION THREAD

**Discussion by kleeedolinux** [Vitor1-1Santana/CMSPTasksDoer#6] [Ideas]
Bom, a maioria das pessoas que utilizarem isto é pra ganahr nota, porém os professores não conseguem ver quanto você acertou ou não (exceto o que for de escrever como redação). Então eu pensei, que tal você adicionar uma opção pra que ele não precse necessariamente acertar as respostas corrgindo assim alguns problemas.

**Comment by Vitor1-1Santana** [comment]
Realmente, na maioria das vezes os professores não ligam para a taxa de acerto nas tarefas. Fiz este programa tentar acertar mesmo assim, para valer a pena a utilidade dele, o que este programa faz, pode ser feito manualmente por todo aluno, e as vezes até com mais precisão. Se este programa chutasse aleatoriamente todas as questões, iria valer a pena fazer algumas atividades manualmente, principalmente nas atividades que tem como você verificar se uma questão está correta.

Mas acho que devo criar uma opção para usuários que não liguem se o programa acerte ou não. A minha ideia é o seguinte:

Ter 3 opções.

1º Opção vai ser o comportamento normal do programa, ele vai tentar acertar as questões, e as que ele não conseguir ou não puder, ele irá salvar a tarefa como rascunho para o usuário fazer manualmente, assim minimizando os "danos" para quem é preocupado com acertos.

2° Opção, o programa irá ter um comportamento parecido com o da primeira opção, mas nas tarefas com questões que não tem o botão verficar, ele irá chutar aleatoriamente as respostas e não salvará como rascunho.

3º Opção, o programa não vai tentar acertar nenhuma questão, irá tentar chutar tudo mesmo que ele consiga verificar se a resposta está correta ou não.

O que acha?

**Reply by kleeedolinux** [reply]
Apoio

**Comment by Vitor1-1Santana** [comment]
Vou criar um issue disso. A próxima build com estas alterações pode sair hoje, ou algum dia semana que vem.
